src/Artifacts.jl

```diff
@@ -17,10 +18,210 @@ end
 """
     soil_params_artifact_path(; context)
 
-Return the path to the folder that contains the soil parameters
+Return the path to the folder that contains the soil parameters.
 """
 function soil_params_artifact_folder_path(; context = nothing)
     return @clima_artifact("soil_params_Gupta2020_2022", context)
 end
 
+"""
+    experiment_fluxnet_data_path(
+        site_ID,
+        data_link,
+        dir;
+        context = nothing,
+    )
+
+Return the path to the file that contains a year of fluxnet data
+corresponding to a `site_ID` in the set (US-MOz, US-NR1, US-Ha1, US-Var).
+
+Site publications and licenses:
+US-Ha1: 
+
+Urbanski, S., Barford, C., Wofsy, S., Kucharik, C., Pyle, E., Budney, J., McKain, K., Fitzjarrald, D., Czikowsky, M., Munger, J. W. 2007. Factors Controlling CO2 Exchange On Timescales From Hourly To Decadal At Harvard Forest, Journal Of Geophysical Research, 112:G2, . 
+
+AmeriFlux FLUXNET: https://doi.org/10.17190/AMF/1871137
+Citation: J. William Munger (2022), AmeriFlux FLUXNET-1F US-Ha1 Harvard Forest EMS Tower (HFR1), Ver. 3-5, AmeriFlux AMP, (Dataset). https://doi.org/10.17190/AMF/1871137
+
+AmeriFlux CC-BY-4.0 Policy
+
+US-MOz
+
+Gu, L., Pallardy, S. G., Yang, B., Hosman, K. P., Mao, J., Ricciuto, D., Shi, X., Sun, Y. 2016. Testing a Land Model In Ecosystem Functional Space via a Comparison of Observed and Modeled Ecosystem Responses to Precipitation Regimes and Associated Stresses in a Central U.S. Forest, Journal Of Geophysical Research: Biogeosciences, 121:7, 1884-1902. 
+
+AmeriFlux FLUXNET: https://doi.org/10.17190/AMF/1854370
+Citation: Jeffrey Wood, Lianhong Gu (2021), AmeriFlux FLUXNET-1F US-MOz Missouri Ozark Site, Ver. 3-5, AmeriFlux AMP, (Dataset). https://doi.org/10.17190/AMF/1854370
+
+AmeriFlux CC-BY-4.0 Policy
+
+US-NR1
+
+Burns, S. P., Blanken, P. D., Turnipseed, A. A., Hu, J., Monson, R. K. 2015. The Influence Of Warm-Season Precipitation On The Diel Cycle Of The Surface Energy Balance And Carbon Dioxide At A Colorado Subalpine Forest Site, Biogeosciences, 12:23, 7349-7377. 
+
+AmeriFlux FLUXNET: https://doi.org/10.17190/AMF/1871141
+Citation: Peter D. Blanken, Russel K. Monson, Sean P. Burns, David R. Bowling, Andrew A. Turnipseed (2022), AmeriFlux FLUXNET-1F US-NR1 Niwot Ridge Forest (LTER NWT1), Ver. 3-5, AmeriFlux AMP, (Dataset). https://doi.org/10.17190/AMF/1871141
+
+AmeriFlux CC-BY-4.0 License
+
+US-Var
+
+AmeriFlux FLUXNET: https://doi.org/10.17190/AMF/1993904
+Citation: Siyan Ma, Liukang Xu, Joseph Verfaillie, Dennis Baldocchi (2023), AmeriFlux FLUXNET-1F US-Var Vaira Ranch- Ione, Ver. 3-5, AmeriFlux AMP, (Dataset). https://doi.org/10.17190/AMF/1993904
+
+AmeriFlux CC-BY-4.0 License
+"""
+function experiment_fluxnet_data_path(
+    site_ID,
+    data_link,
+    dir;
```

Comment:
I think that `dir` should not be passed. Instead dir should be `@__DIR__` of this file. With this, I think all the artifacts will live in the same Artifacts.toml

Reply:
this seemed like it worked (it put the experiment artifacts in the src/Artifacts.toml)